no-ui-api-access.md

```diff
@@ -0,0 +1,39 @@
+# Access the API without a UI
```

Comment:
Je suis pas super satisfait de mon nom de fichier, mais je sais pas trop c'est quoi qui serait le meilleur non 🤔

Reply:
Si tes down, je ferais un autre folder au root `non-browser-apps`. 

Le titre ça pourrait être `Browserless access to the API`.